packages/visualizations/addon/serializers/pie-chart.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+import Ember from 'ember';
+import VisualizationSerializer from 'navi-visualizations/serializers/visualization';
+import { parseMetricName } from 'navi-data/utils/metric';
+
+const { isPresent, get } = Ember;
+
+export default VisualizationSerializer.extend({
+  /**
+   * Normalizes payload so that it can be applied to models correctly
+   * @param type - class type as a DS model
+   * @param visualization - json parsed object
+   * @return {Object} normalized payload
+   */
+  normalize(type, visualization) {
+    if(visualization) {
+      let config = get(visualization, 'metadata.series.config');
+
+      if(isPresent(config)) {
+        let metric = get(config, 'metric');
+
+        if(typeof metric === 'string') {
+          let metricObj = parseMetricName(metric);
+
+          visualization.metadata.series.config.metric = metricObj;
```

Comment:
Can we reduce the number of lines here? :bowtie: 
`visualization.metadata.series.config.metric = parseMetricName(metric)`